src/site/content/en/blog/browser-flags/index.md

```diff
@@ -0,0 +1,82 @@
+  This article explains how to do this in the various Chromium derivatives like Google Chrome, Microsoft Edge, and others.
+authors:
+  - thomassteiner
+date: 2021-05-17
+description: |
+  For some of the new APIs we introduce in Chromium, you need to set a browser flag for testing.
+  This article explains how to do this in the various Chromium derivatives like Google Chrome, Microsoft Edge, and others.
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/NcSd5vAM9zMrbQqwVZhl.jpg
+alt: Various party flags.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+---
+
+[Chromium](https://www.chromium.org/) is an open-source browser project that aims to build a safer,
+faster, and more stable way for all users to experience the web. A lot of web browsers are built on
+Chromium, including the popular browsers [Google Chrome](https://www.google.com/chrome/) by
+[Google](https://www.google.com/), [Microsoft Edge](https://www.microsoft.com/en-us/edge) by
+[Microsoft](https://www.microsoft.com/), [Opera Web Browser](https://www.opera.com/) by
+[Opera](https://www.opera.com/about), and
+[many others](<https://en.wikipedia.org/wiki/Chromium_(web_browser)#Browsers_based_on_Chromium>).
+
+## The `chrome://` scheme
+
+Google Chrome has since the beginning supported a special scheme called `chrome://` for accessing
+browser-internal settings or features. You can see the (almost) full list by putting
+`chrome://chrome-urls` into the URL bar. The most common such special URL is `chrome://flags`.
+
+## Setting browser flags
+
+For some of the [new APIs](/tags/capabilities/) we introduce in Chromium, you need to set a browser
+flag for testing. You guessed it, `chrome://flags` is where this happens. The most popular flag we
+ask you to set is `chrome://flags/#enable-experimental-web-platform-features`, which, as the name
+suggests, enables experimental web platform features.
+
+{% Aside 'warning' %} As the name suggests, experimental web platform features are _experimental_.
+We do not recommend you set flags on your daily production browser. Instead, we prefer you to use a
+development browser version like the Beta, Dev, or Canary channel for your development needs where
+you can set flags as required, and the Stable channel for everything else. {% endAside %}
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/vtpmqTzfmsru5ZhxNuGt.png", alt="Toggling the 'experimental web platform features' flag.", width="800", height="125" %}
+
+{% Aside 'key-term' %} Browser flags are distinct from [origin trials](/origin-trials/).
+
+Origin trials are set by website owners and opt in your local browser to a given feature. Only
+features deemed safe for testing with real users are available for origin trials.
+
+Browser flags are set by you and opt in your local browser to a given feature. Not all features that
+are available behind a flag are ready for prime time. {% endAside %}
```

Comment:
I'm not crazy about 'prime time' as a metaphor for flags. A feature behind a flag is like like an unaired [television pilot](https://www.google.com/search?q=television+pilot&oq=television+pilot&aqs=chrome..69i57j0l9.11791j0j15&sourceid=chrome&ie=UTF-8). In contrast a show that airs at 11:00 PM may not be prime time, but it's still production. I'm worried that readers will take the wrong meaning from this.

Reply:
Went with "ripe for production".